Makefile

```diff
@@ -15,14 +15,16 @@ dependencies:
 	# pin version due to https://github.com/python-poetry/poetry/issues/7184
 	pip install poetry==1.2.2 --quiet --ignore-installed
 	poetry --version
-	poetry install --no-root
+	poetry install
```

Comment:
turns out --no-root prevents installing Hokusai itself.

Reply:
right... it's a slightly strange name for that option. I assume `make dependencies` was designed to only install the dependencies, primarily for the CI/testing use case. We generally build the project package when running tests, not sure if your tests assume that. It's relevant since some module paths may differ based on the assumption.

Reply:
ah, thanks for pointing that out. since the step is named `dependencies`, `--no-root` seems appropriate. i should perhaps rename it, or create a separate `make` step for the integration tests.